test/evm/hardhat/chain-specific-spokepools/Arbitrum_SpokePool.ts

```diff
@@ -138,4 +173,59 @@ describe("Arbitrum Spoke Pool", function () {
     expect(l2GatewayRouter[functionKey]).to.have.been.calledOnce;
     expect(l2GatewayRouter[functionKey]).to.have.been.calledWith(dai.address, hubPool.address, amountToReturn, "0x");
   });
+
+  it("Bridge tokens to hub pool correctly using the OFT messaging for L2 USDT token", async function () {
+    const l2UsdtSendAmount = BigNumber.from("1234567");
+    const { leaves, tree } = await constructSingleRelayerRefundTree(
+      l2UsdtContract.address,
+      await arbitrumSpokePool.callStatic.chainId(),
+      l2UsdtSendAmount
+    );
+    await arbitrumSpokePool.connect(crossDomainAlias).relayRootBundle(tree.getHexRoot(), mockTreeRoot);
+
+    const oftNativeFee = BigNumber.from(1e9 * 200_000); // 1 GWEI gas price * 200,000 gas cost
```

Comment:
Same comment as above, this isn't neccessarily safe. Do `toWeiWithDecimals("1", 9).mul(200_000)`